docs/source/pythonapi/index.rst

```diff
@@ -4,29 +4,59 @@
 Python API
 ==========
 
---------
-Contents
---------
+OpenMC includes a rich Python API that enables programmatic pre- and
+post-processing. The easiest way to begin using the API is to take a look at the
+example Jupyter notebooks provided. The full API documentation serves to provide
```

Comment:
Perhaps include a link to the [Jupyter Project](https://jupyter.org/)? It might also be worthwhile to direct the reader to some resources on how to get started with Jupyter and IPython. Some ideas that come to mind are the following:

* [IPython Documentation](http://nbviewer.ipython.org/github/ipython/ipython/blob/3.x/examples/Index.ipynb) - includes tutorials
* [Code Academy](https://www.codecademy.com/tracks/python) - my novice freshman UROPs have found this to be useful as new programmers 
* [Notebook Gallery](http://nb.bianp.net/sort/views/)

Perhaps some true Pythonistas (unlike myself) such as @scopatz may refer us to some even better resources.